packages/marshal/src/marshal.js

```diff
@@ -345,8 +345,14 @@ function assertCanBeRemotable(val) {
   assert(!Array.isArray(val), X`Arrays cannot be pass-by-remote`);
   assert(val !== null, X`null cannot be pass-by-remote`);
 
-  const keys = ownKeys(val);
+  const descs = getOwnPropertyDescriptors(val);
+  const keys = ownKeys(descs); // enumerable-and-not, string-or-Symbol
   keys.forEach(key => {
+    // @ts-ignore
```

Comment:
I see from the CI output that this `@ts-ignore` isn't working for you anyway. I suspect it might if you move it down a line, but see below instead.